shows_app/src/app/components/shared/SideBar/SidebarNavigation.tsx

```diff
@@ -1,57 +1,141 @@
 "use client";
 
+import React from "react";
 import {
   Box,
   Button,
-  Divider,
   Flex,
+  Img,
   Spacer,
   Stack,
-  Text
+  Text,
+  useBreakpointValue,
+  useDisclosure,
+  Drawer,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerHeader,
+  DrawerBody,
+  IconButton
 } from "@chakra-ui/react";
 import NextLink from "next/link";
 import { useRouter } from "next/navigation";
+import { CloseIcon, HamburgerIcon } from "@chakra-ui/icons";
 
 export const SideBarNavigation = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
   const router = useRouter();
 
   const handleLogout = () => {
     localStorage.removeItem("authToken");
     router.push("/login");
   };
 
+  const sidebarWidth = useBreakpointValue({
```

Comment:
We usually add this directly on our component:
`<Box width={{base: '100%, md: '300px'}}`